website/docs/contributing/developer-guide.md

```diff
@@ -4,25 +4,19 @@ title: Developer guide
 
 ## Introduction {#introduction}
 
-This repository contains two main parts. The backend and the frontend of unleash. The backend is a Node.js application that is built using TypeScript. The frontend is a React application that is built using TypeScript. The backend specific code can be found in the `src` lib folder. The frontend specific code can be found in the `frontend` folder.
+This repository consists of two main parts: the backend and frontend of Unleash. The backend is a Node.js application built with TypeScript, while the frontend is a React application also built with TypeScript. You can find code specific to the backend in the `src` lib folder and code specific to the frontend in the `frontend` folder.
 
 ## Development philosophy
 
-The development philosophy at unleash is centered at delivering high quality software. We do this by following a set of principles that we believe will help us achieve this goal. We believe that these principles will also help us deliver software that is easy to maintain and extend, serving as our north star.
+The development philosophy at Unleash is centered on delivering high-quality software. We achieve this by following a set of principles that we believe will also make the software easy to maintain and extend, serving as our guide.
 
-We believe that the following principles will help us achieve our goal of delivering high quality software:
+We believe that the following principles are essential in achieving our goal:
 
-* We test our code always
+- Testing code always: Software engineering is challenging, and it's crucial to take every precaution necessary to avoid introducing bugs. Thus, testing is an effective way to accomplish this aim. We prefer automation over manual testing.
+ 
+- Writing code that is easy to understand and maintain: Code serves as language – a means of communicating intent with readers efficiently. Therefore, writing clean code matters significantly for us because it keeps our codebase manageable while maintaining speed in the long run.
```

Comment:
Unsure if this rewrite is better. Thoughts?

Reply:
I'm fine with either. I like the old one, but I think this one works well too.

Reply:
Completely neutral on this one. I don't think it's worse but it's consistent with the tone of other parts that have been rewritten

Reply:
I think the old one wins here. The old version has more heart while this feels like just another generic AI summary.